odl/operator/operator.py

```diff
@@ -470,10 +470,10 @@ def __init__(self, domain, range, linear=False):
             raise TypeError('`range` {!r} is not a `Set` instance'
```

Comment:
Apparently we overwrite these in some other class (the functionspacevector) so they seem to be part of the subclass API.

This is kindoff the exact reason I'm doing this change, so things like this is obvious (and it should force us to document these things).